### PR TITLE
Release/0.2.0

### PR DESCRIPTION
**Pull request by jbeemster**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -100,51 +79,61 @@ resource "aws_iam_policy" "iam_policy" {
...
+      },
+      {
+        Effect = "Allow",
+        Action = [
+          "sqs:DeleteMessage",
+          "sqs:GetQueueUrl",
+          "sqs:ListQueues",
+          "sqs:ChangeMessageVisibility",
+          "sqs:ReceiveMessage",
+          "sqs:SendMessage"
```
Comment (istreeter) [inline main.tf:110]: I wonder why `sqs:SendMessage` is in this list.  I cannot think why the loader needs this.

File: templates/user-data.sh.tmpl [hunk 2]
```diff
@@ -45,7 +25,8 @@ sudo docker run \
   --log-opt max-file=5 \
 %{ endif ~}
   -v $${CONFIG_DIR}:/snowplow/config \
-  -e 'JAVA_OPTS=-Dorg.slf4j.simpleLogger.defaultLogLevel=info' \
+  -e 'JAVA_OPTS=${java_opts}' \
```
Comment (istreeter) [inline templates/user-data.sh.tmpl:28]: Have we discussed yet about using the distroless images for open source terraform?  If you did want to switch to distroless, then you would need to change `JAVA_OPTS` to `JDK_JAVA_OPTIONS`.

**Review by istreeter — APPROVED** [review]

File: templates/user-data.sh.tmpl (hunk 2 above)
Comment (jbeemster) [reply templates/user-data.sh.tmpl:28]: We haven't - this round of changes is focused on getting launch_templates + IMDSv2 in place + auto-scaling rules.  Will leave moving to distroless to another time!

File: main.tf (hunk 1 above)
Comment (jbeemster) [reply main.tf:110]: Good point!  If the code base never invokes this should be safe to remove.  Equally feels like a fairly safe one to leave in?

**Pull request merged**